go/vt/vttablet/onlineddl/executor.go

```diff
@@ -2703,8 +2725,12 @@ func (e *Executor) OnSchemaMigrationStatus(ctx context.Context, uuidParam, statu
 	if eta, err := strconv.ParseInt(etaParam, 10, 64); err == nil {
 		etaSeconds = eta
 	}
+	var rowsCopied int64 = 0
```

Comment:
Weird that the linter is not complaining about this. I thought we had a lint for not zero-initializing variables that are already zero-initialized.

Reply:
Agreed, weird. I'll clean it nonetheless.